ic-factory/src/api.rs

```diff
@@ -18,6 +18,8 @@ pub trait FactoryCanister: Canister + Sized + PreUpdate {
     // default definitions of the `FactoryCanister` will use `ic-factory`
     // state, instead of the state of the factory canister, that
     // implements this trait.
+    // TODO: remove default implementation since the user may forget
```

Comment:
Is there a reason the default impl wasn't removed?

Reply:
Yeah, may break the downstream and I didn't want for this PR to relate to it

Reply:
Is there a tracking issue for this so it's not forgotten?

Reply:
I thought there was 
https://infinityswap.atlassian.net/browse/CPROD-1056